.ci_scripts/update_teams

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+# Expected to be run in a TravisCI context.
+
+set -e
+
+if [ -n "$GH_TOKEN" ]; then
+    conda execute -v ./scripts/update_teams.py ./feedstocks
```

Comment:
It still concerns me that the update_teams is based off of this repo, rather than using the `conda_smithy.feedstocks` submodule.